kickoff/firefoxdetails.py

```diff
@@ -0,0 +1,784 @@
+from firefoxversions import firefoxversions as versions
+
+primary_builds = {
```

Comment:
I feel like this (and some other) data structures could be done in a better way, but I'm having a lot of trouble understanding the overall structure. I think it would help a lot if you could explain a couple of things:
1) What data are we currently missing from Ship It that we need to generate the JSON files.
2) Which data (existing and new) goes with which endpoints?